ui/src/app/shared/components/layout/layout.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import {Sidebar} from '../../../sidebar/sidebar';
+
+require('./layout.scss');
+
+export interface LayoutProps {
```

Comment:
Is the layout component the main layout or sidebar layout?

Reply:
Layout is a main layout, copied from `argo-ui`